src/conf.py

```diff
@@ -325,9 +376,9 @@
 
 # One entry per manual page. List of tuples
 # (source start file, name, description, authors, manual section).
-man_pages = [
-    ('index', 'cellml', u'CellML 2.0 Documentation', [u'CellML Editors'], 1)
-]
+#man_pages = [
+#    ('index', 'cellml', u'CellML 2.0 Documentation', [u'CellML Editors'], 1)
+#]
```

Comment:
Think about removing this, if not needed.

Reply:
These are usually left in to mark what could be, pretty standard for Sphinx conf.py files. Otherwise we might be removing everything that is not in use.

Reply:
I like things to be cleaned, but your call. (I mean, it's easy to put back, if ever needed.)